api/src/main/java/org/ehcache/spi/loaderwriter/WriteBehindConfiguration.java

```diff
@@ -23,70 +23,80 @@
  */
 public interface WriteBehindConfiguration extends ServiceConfiguration<WriteBehindDecoratorLoaderWriterProvider> {
   /**
-   * the minimum number of seconds to wait before writing behind
+   * The minimum number of seconds to wait before writing behind.
+   *
+   * Lower than or equal to {@see getMaxWriteDelay}.
    *
    * @return Retrieves the minimum number of seconds to wait before writing behind
    */
   int getMinWriteDelay();
 
   /**
-   * the maximum number of seconds to wait before writing behind
+   * The maximum number of seconds to wait before writing behind.
+   *
+   * Greater than or equal to {@see getMinWriteDelay}
    *
    * @return Retrieves the maximum number of seconds to wait before writing behind
    */
   int getMaxWriteDelay();
 
   /**
-   * the maximum number of write operations to allow per second.
+   * The maximum number of write operations to allow per second.
+   *
+   * {@code 0} means no limit.
    *
    * @return Retrieves the maximum number of write operations to allow per second.
    */
   int getRateLimitPerSecond();
 
   /**
-   * whether write operations should be batched
+   * Whether write operations should be batched.
    *
    * @return Retrieves whether write operations should be batched
    */
   boolean isWriteBatching();
 
   /**
-   * write coalescing behavior
+   * Whether write operations can be coalesced.
    *
    * @return Retrieves the write coalescing behavior is enabled or not
    */
   boolean isWriteCoalescing();
 
   /**
-   * the size of the batch operation.
+   * The recommended size of a batch of operations.
+   *
+   * Real batch size will be influenced by arrival frequency of operations and max write delay.
    *
    * @return Retrieves the size of the batch operation.
    */
   int getWriteBatchSize();
 
   /**
-   * the number of times the write of element is retried.
+   * The number of times the write of a mapping is retried in case it fails.
```

Comment:
I don't think this sentence means what you want it to:
> The number of times the write of a mapping is retried in case it fails.
versus
> The number of times the write of a mapping will be retried in the case of failure.